tests/test_types.py

```diff
@@ -61,16 +60,16 @@ def test_is_not_valid(self):
         assert not vatin.is_valid()
 
     @patch('vies.types.Client')
-    def test_raises_when_suds_web_fault(self, mock_client):
-        """Raise an error if suds raises a WebFault."""
+    def test_raises_when_zeep_web_fault(self, mock_client):
+        """Raise an error if zeep raises a WebFault."""
```

Comment:
The doc string is now misleading. Because zeep doesn't raise a `WebFault` exception.

Reply:
```suggestion
        """Raise an error if zeep raises an exception."""
```